### DISCUSSION TITLE
Port to Edge Chromium

### DISCUSSION THREAD

**Discussion by piggov** [furybee/chrome-tab-modifier#278] [Ideas]
I know usually Edge can open Chrome extensions. But my organization is very strict on which store the extensions come from. So if it's not on the [Edge Add-ons](https://microsoftedge.microsoft.com/addons/Microsoft-Edge-Extensions-Home) page, it's not allowed.

Is it possible to port the extension to Edge and publish it in the Microsoft Add-ons page?

**Comment by sylouuu** [comment]
Hi @Bonnev,

Tab Modifier is now available on [Microsoft Edge Addons](https://microsoftedge.microsoft.com/addons/detail/tab-modifier/edfkealmjdmkfimdjmiapmnakepfilkp)

Bests

**Comment by piggov** [comment]
@sylouuu Wow, thanks man, I really appreciate it!
